web-lib-funcs.pl

```diff
@@ -11001,16 +11001,15 @@ =head2 unix_crypt(password, salt)
 sub unix_crypt
 {
 my ($pass, $salt) = @_;
-return "" if ($salt !~ /^[a-zA-Z0-9\.\/]{2}/);   # same as real crypt
-my $rv = eval "crypt(\$pass, \$salt)";
-my $err = $@;
-return $rv if ($rv && !$@);
+return "" if ($salt !~ /^[\$a-zA-Z0-9]{2}/);   # same as real crypt
+my $rv = eval { crypt($pass, $salt) };
```

Comment:
I think the use of eval "" should stay, as there are some rare systems on which `crypt` doesn't work.

Reply:
Yes, but I didn't remove `eval`. Also, I really doubt those systems are still around.

Reply:
Ok, I think this will be fine then

Reply:
Yeah, it is. Tested it.